requirements.txt

```diff
@@ -6,7 +6,7 @@ requests==2.25.1
 uvicorn==0.13.4
 pymongo==3.11.3
 mongomock==3.22.1
-django==3.1.8
+django==3.2
```

Comment:
I think we can probably remove this now; the django tests won't run unless we install this in the CI, and all django functionality has been deprecated for a while. Perhaps it is better to remove this in #750 instead of here.

Reply:
Indeed. That was my thought.
We remove both Django support, Python 3.6 support, etc. A standard clean-up for v0.15 :)